extensions/duck-duck-go-search/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # DuckDuckGo Search Changelog
 
+## [Version 1.1] - 2023-04-10
+
+- NEW: show favicons for the hardcoded bangs
+
```

Comment:
Add `Limited support for [DuckDuckGo !Bangs](https://duckduckgo.com/bangs)` or something like those lines.

Reply:
> Also, looks like there is no setLoading(false) after searching. I can't try the updated extensions right now, but it seems wrong to not have that line?

This is done after the results are there, see line 77. I have implemented all other suggestions. :)